res/css/views/settings/encryption/_ChangeRecoveryKey.pcss

```diff
@@ -68,5 +68,28 @@
         display: flex;
         flex-direction: column;
         gap: var(--cpd-space-8x);
+
+        .mx_KeyForm_password {
+            > input[name="recoveryKey"] {
+                /*
+                 * From figma https://www.figma.com/design/qTWRfItpO3RdCjnTKPu4mL/Settings?node-id=375-77506&t=d82NdRBDoKsUe1C9-4
+                 */
+                height: 70px;
+                padding: var(--cpd-space-3x) var(--cpd-space-3x) var(--cpd-space-3x) var(--cpd-space-4x);
+                border: var(--cpd-border-width-1) solid;
+                border-radius: 8px;
+                margin: 0px;
+            }
+
+            > button {
+                /*
+                 * See figma https://www.figma.com/design/qTWRfItpO3RdCjnTKPu4mL/Settings?node-id=375-77506&t=d82NdRBDoKsUe1C9-4
+                 * Avoid stretching the hide/show symbol to the height of the input, and centre it vertically.
+                 */
+                height: 24.5px;
+                padding: var(--cpd-space-1x);
+                align-self: center;
+            }
```

Comment:
bit surprised to find this is so different from the CSS in https://github.com/element-hq/element-web/pull/30471. Indeed, I'd expect us to have a single component that is used in both places. Isn't it basically the same control?

Reply:
Ugh. Why do we need different components depending on whether or not we are in a form?

Reply:
seems like *somewhere* there should be some shared implementation, but fine, I'll stop pulling at this thread.